shopelectro/tests/tests_js_analytics.py

```diff
@@ -59,7 +74,48 @@ def test_google_ecommerce_purchase(self):
             )
         self.assertEqual(send, ['ecommerce:send', None])
 
-    # @todo #771:60m Test Yandex eCommerce
+
+@tag('slow')
+@helpers.disable_celery
+@override_settings(DEBUG=True, INTERNAL_IPS=tuple())
+class YandexEcommerce(Ecommerce):
+
+    fixtures = ['dump.json']
+
+    # @todo #785:60m Test Yandex ecommerce goals.
+    #  Here goals are left to test:
+    #  - onCartClear from cart
+    #  - onProductAdd from catalog, product and order pages
+    #  - onProductRemove from cart and order page
+    #  - onProductDetail from product page
+
+    def reached_goals(self):
+        return self.browser.execute_script('return window.dataLayer.results;')
```

Comment:
we should somehow document goals structure. Pdd subtask might be good for it

Reply:
The structure already documented in yandex docs